components/templates/Homepage/index.tsx

```diff
@@ -50,6 +50,45 @@ export default function Homepage() {
           </p>
         </div>
       </section>
+
+      <section data-testid="section_two">
+        <div>
+          <h2>
```

Comment:
Component breakdown lists this as an h3!

Reply:
This is an issue with the component breakdown. Made corrections and uploaded new version to google drive.